go.mod

```diff
@@ -23,12 +24,15 @@ require (
 	gotest.tools v2.2.0+incompatible
 	k8s.io/api v0.31.2
 	k8s.io/apimachinery v0.31.2
-	k8s.io/client-go v0.31.1
+	k8s.io/client-go v12.0.0+incompatible
```

Comment:
This comes from the victoria-metrics package. To fix this I have pinned the dependencies below in the `replace` block